### DISCUSSION TITLE
Error while installing desktop app build with AIR SDK 51.1.1.2 and 51.1.1.1

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#3397] [General]
While installing the app below is the error image.

![image](https://github.com/user-attachments/assets/3edbab43-7cc8-495a-890f-959d4c6c3385)
@ajwfrost I already downloaded the updated AIR runtime from https://airsdk.harman.com/runtime having runtime version 51.0.1.3

Is new runtime is still not share on download page?

**Comment by jeuxdaily** [comment]
are you solve your problem

**Comment by balaedit** [comment]
> are you solve your problem

Not yet

**Reply by jeuxdaily** [reply]
not yet I'm working for

**Comment by ajwfrost** [comment]
Ah - yes, so we would need to push up the 51.1 runtime for these new apps to work properly. It might be best for us to wait until 51.1.1.3 which is hopefully landing around the end of this week..
(Still better to use a captive runtime bundle though, rather than relying on the shared runtime!)

**Comment by balaedit** [comment]
> Ah - yes, so we would need to push up the 51.1 runtime for these new apps to work properly. It might be best for us to wait until 51.1.1.3 which is hopefully landing around the end of this week..
> (Still better to use a captive runtime bundle though, rather than relying on the shared runtime!)

I encountered an issue where, despite publishing with the latest AIR SDK and installing the most recent AIR runtime, the system prompted me for an update. This discrepancy was unexpected. Thank you for your attention to this matter.

**Comment by balaedit** [comment]
> Ah - yes, so we would need to push up the 51.1 runtime for these new apps to work properly. It might be best for us to wait until 51.1.1.3 which is hopefully landing around the end of this week..
> (Still better to use a captive runtime bundle though, rather than relying on the shared runtime!)

Received the latest runtime with SDK. Will test. Thanks
